src/main/java/com/esri/samples/scene/get_elevation_at_a_point/GetElevationAtAPointSample.java

```diff
@@ -0,0 +1,179 @@
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.scene.get_elevation_at_a_point;
+
+import java.util.concurrent.ExecutionException;
+
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol;
+import javafx.application.Application;
+import javafx.geometry.Point2D;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.input.MouseButton;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.concurrent.ListenableFuture;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.geometry.Polyline;
+import com.esri.arcgisruntime.geometry.PolylineBuilder;
+import com.esri.arcgisruntime.mapping.ArcGISScene;
+import com.esri.arcgisruntime.mapping.ArcGISTiledElevationSource;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Surface;
+import com.esri.arcgisruntime.mapping.view.Camera;
+import com.esri.arcgisruntime.mapping.view.Graphic;
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.mapping.view.LayerSceneProperties;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+import com.esri.arcgisruntime.symbology.TextSymbol;
+
+public class GetElevationAtAPointSample extends Application {
+
+    private SceneView sceneView;
+    private GraphicsOverlay graphicsOverlay;
+
+    @Override
+    public void start(Stage stage) {
+
+        try {
+            // create stack pane and application scene
+            StackPane stackPane = new StackPane();
+            Scene fxScene = new Scene(stackPane);
+            fxScene.getStylesheets().add(getClass().getResource("/css/style.css").toExternalForm());
+
+            // set title, size, and add JavaFX scene to stage
+            stage.setTitle("Get Elevation At A Point Sample");
+            stage.setWidth(800);
+            stage.setHeight(700);
+            stage.setScene(fxScene);
+            stage.show();
+
+            // create a scene and add a basemap to it
+            ArcGISScene scene = new ArcGISScene();
+            scene.setBasemap(Basemap.createImagery());
+
+            // add the SceneView to the stack pane
+            sceneView = new SceneView();
+            sceneView.setArcGISScene(scene);
+            stackPane.getChildren().add(sceneView);
+
+            // add base surface for elevation data
+            Surface surface = new Surface();
+            surface.getElevationSources().add(new ArcGISTiledElevationSource("http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"));
+            scene.setBaseSurface(surface);
+
+            // create a point symbol to mark where elevation is being measured
+            SimpleLineSymbol lineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, 0xFFFF0000, 3.0f);
```

Comment:
Either call this `elevationLineSymbol` or the other one just `textSymbol` for consistency.